api/envoy/extensions/filters/http/ext_proc/v3/ext_proc.proto

```diff
@@ -265,6 +280,12 @@ message ExternalProcessor {
   google.protobuf.Duration deferred_close_timeout = 19;
 }
 
+// HttpService is used for raw HTTP communication between the filter and the external processing service.
+message HttpService {
```

Comment:
Should we be moving to the core HttpService message from now on? I know there is prior art in ext_authz, but would be good to break from N HttpService definitions.

Reply:
Yeah, I thought about that.  But, the HttpService defined for ext_authz has a lot of ext_authz specific fields in it:
https://github.com/envoyproxy/envoy/blob/172679e914f94e529679b88b67665a888858b1df/api/envoy/extensions/filters/http/ext_authz/v3/ext_authz.proto#L359.

Reply:
Sure, we can use this message.  But, I would still want to keep the ext_proc wrapper message just in case we want to add some more ext_proc specific fields into it in the future.

Reply:
OK, that sounds like a good idea.